### PR TITLE
Fix inconsistent `device_id`

### PR DESCRIPTION
**Pull request by robbevp**
We use `device_id` inconsistently thoughout the different methods.

### CODE REVIEW

File: src/api_module.js [hunk 1]
```diff
@@ -41,12 +41,12 @@ export class ApiModule {
 
     if (routes.includes("read")) {
       this.read = async function (auth, id, retryOptions = {}) {
-        const request = new Request(`${this.path}/${id}`, {
+        const request = new Request(`${this.path}/${id || ""}`, {
```
Comment (chvp) [inline src/api_module.js:44]: This doesn't seem related? I also don't think we should want to support passing an empty id?

Comment (robbevp) [reply src/api_module.js]: this slipped in, removed from this PR

**Review by chvp — APPROVED** [review]
